sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Framework/UcfgVerifier.cs

```diff
@@ -0,0 +1,119 @@
+﻿/*
+* SonarAnalyzer for .NET
+* Copyright (C) 2015-2018 SonarSource SA
+* mailto: contact AT sonarsource DOT com
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this program; if not, write to the Free Software Foundation,
+* Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+extern alias csharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using csharp::SonarAnalyzer.ControlFlowGraph.CSharp;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using SonarAnalyzer.Protobuf.Ucfg;
+
+namespace SonarAnalyzer.UnitTest.Security.Framework
+{
+    internal static class UcfgVerifier
+    {
+        public static UCFG GetUcfgForMethod(string code, string methodName)
```

Comment:
GetUcfgForMethod/Constructor could be further deduplicated - CFG and UCFG generation could be moved to VerifyInstructions...

Reply:
It cannot yet. Some tests still relies on this.